tests/at_functional_test/test/scan_verb_test.dart

```diff
@@ -120,18 +120,31 @@ void main() {
 
     ///UPDATE VERB
     await socket_writer(
-        socketFirstAtsign!, 'update:public:verifyingTTL$firstAtsign Working?');
+        socketFirstAtsign!, 'update:ttl:3000:ttlkey.me$firstAtsign 1245');
```

Comment:
Can you modify this test so that the entity is 'ttlKey.me' (i.e. capitalize the k) and modify the expects accordingly? Would be good to be asserting that in the scan response, the entity is in lowercase

Reply:
sure

Reply:
Updated the test with a capital K. But the scan response doesn't show up the key. Raised a bug for the same.
[bug](https://github.com/atsign-foundation/at_server/issues/1216)

Reply:
You haven't changed the expects to match :-)

Reply:
Oh I see, you didn't commit the changes

Reply:
Yes @gkc . I haven't pushed my changes because the test will fail in actions